content/docs/programs/immunizations.md

```diff
@@ -0,0 +1,131 @@
++++
+title = "Immunizations"
+description = "Immunization Programs"
+date = 2022-05-17T16:20:00+00:00
+updated = 2022-05-17T16:20:00+00:00
+draft = false
+weight = 62
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+Choose `Programs` > `Immunizations` in the navigation panel.
+
+![goto: immunizations](/docs/programs/images/goto_immunizations.png)
+
+You will be presented with a list of Immunization Programs:
+
+![immunizations list](/docs/programs/images/immunizations.png)
+
+The list of Immunization Programs is divided into 2 columns:
+
+| Column              | Description                                                      |
+| :------------------ | :--------------------------------------------------------------- |
+| **Program Name**    | The name of the Immunization Program                             |
+| **Vaccine Courses** | The Vaccine Courses that are a part of that Immunization Program |
+
+## Adding a new Immunization Program
+
+To add a new Immunization Program, click the `New Program` button in the top right.
+
+![immunizations: new program button](/docs/programs/images/immunizations_new_program_button.png)
+
+This will open a new window, where you can enter the name of the new Immunization Program. Note that all programs must have a unique name.
+
+![immunizations: new program modal](/docs/programs/images/immunizations_new_program_modal.png)
+
+- Click `OK` once you are happy with your changes
+  - You will be redirected to the detail view for your new Immunization Program
+- OR: Click `Cancel` at any time to revert your changes
```

Comment:
```suggestion
-Click `Cancel` to close the window if you've changed your mind
```

or something? there's not much to revert so that didn't sound right..